### DISCUSSION TITLE
One loop places the seam in the wrong spot

### DISCUSSION THREAD

**Discussion by Garryb123** [supermerill/SuperSlicer#2686] [General]
I have seam set to rear but one loop starts in the corner and the little random blob there spoils the otherwise flawless print. Can the seam algorithm be tweaked to stop this?

![image](https://user-images.githubusercontent.com/5060580/162816156-62da8db9-ce1c-4162-93ec-8cbb6d450bab.png)

**Comment by supermerill** [comment]
Can you open an issue with the problematic project file?
